packages/@okta/vuepress-site/docs/guides/device-authorization-grant/main/index.md

```diff
@@ -63,35 +63,35 @@ curl --request POST \
 The device authorization request passes the following parameters:
 
 * `client_id` &mdash; matches the Client ID of the OAuth 2.0 application that you created
-* `scope` &mdash; specifies which access privileges are being requested for access tokens. See [Scopes](/docs/reference/api/oidc/#scopes) for a list of supported scopes.
+* `scope` &mdash; specifies which access privileges are being requested for the access token. See [Scopes](/docs/reference/api/oidc/#scopes) for a list of supported scopes.
 
 **Example response**
 
 ```json
 {
-   "device_code": "f5ec0430-c2fe-4a93-912d-08631f1effed",
-   "user_code": "HMBSRJRC",
-   "verification_uri": "https://${yourOktaDomain}/activate",
-   "expires_in": 600,
-   "interval": 5
+    "device_code": "4ebdb4de-1f8b-4497-be01-ddfaf83c4e9c",
+    "user_code": "MHXTFRPK",
+    "verification_uri": "https://atko-demo.oktapreview.com/activate",
```

Comment:
Why change this? We should stick to using either an example domain or the `${yourOktaDomain}` throughout the guide, shouldn't we?

Reply:
sorry - missed that one - Good catch!